source/util/misc_test_utils.go

```diff
@@ -0,0 +1,257 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package util
```

Comment:
We did move this out of source.go, but it's still all exported functions, which anyone importing conduit-connector-marketo can use. I believe we should make them all unexported. However, we can refactor that in a separate issue. Let me know what you think about that.

Reply:
refactored as test file